src/rustdoc/proposals-and-stabilization.md

```diff
@@ -0,0 +1,70 @@
+# Proposals, Approvals and Stabilization
+It is very common to need to gather feedback and approval when contributing to rustdoc, either
+for permission to proceed with an experiment or refactoring, or when stabilizing a feature. This
+document aims to summarise the various processes that the rustdoc team has for making approval
+decisions and when each should be used.
+
+## Approvals
+There are three mechanisms that the team can use to approve a proposal (not all approval mechanisms
+are suitable for each method of making a proposal - see below):
+
+- r+
+  - A proposal is r+'d when it is approved to be merged.
+  - r+ can only be used to approve a PR.
+- FCP
+  - A final comment period will require sign-off from a majority of the rustdoc team to approve
+    a proposal and then a ten day waiting period.
+  - FCPs can be used to approve any form of proposal.
+
+## Proposals
+There are two ways to propose a change to the rustdoc team. The appropriate choice depends on
+the nature of the proposal, described below.
+
+- Request For Comments (RFC)
+  - RFCs are pull requests to the [`rust-lang/rfcs`][rfcs] repository and are a heavy-weight
+    proposal mechanism, reserved for significant changes.
+  - RFC proposals can only be approved by *FCPs*.
+- Pull Request (PR)
```

Comment:
It would be nice if we had a lightweight process for "is this something the team is interested in" so contributors don't have to waste time throwing together a patch or writing an rfc for something that was always going to be rejected.

basically some equivalent to ACPs.

some people would say that's what IRLO is for, but none of the team is active there so it is unlikely you will get helpful feedback.

Reply:
Fair enough. Adding it.